base-rules/dell-redfish-v117.yaml

```diff
@@ -0,0 +1,104 @@
+Traverse:
+  Root: /redfish/v1
+  Excludes:
+    # Schemas
+    - /JsonSchemas
+    # Non-metric properties
+    - /Accounts
+    - /Certificates
+    - /Jobs
+    - /Logs
+    - /Registries
+    - /Roles
+    - /Sessions
+    - /Settings
+    # Services
+    - /AccountService
+    - /CertificateService
+    - /EventService
+    - /JobService
+    - /LogServices
+    - /SessionService
+    - /TaskService
+    - /TelemetryService
+    - /UpdateService
+    - /LicenseService  # Add v1.17
+    # Sub-metrics included in /Power or /Thermal
+    - /Power/
+    - /Power#/
+    - /Thermal#/
+    - /Assembly#/
+    # Redundant page with extra trailing slash
+    - /redfish/v1/Chassis/$
+
+Metrics:
+  ## Chassis
+  - Path: /redfish/v1/Chassis/{chassis}
+  - Path: /redfish/v1/Chassis/{chassis}/Assembly
+  - Path: /redfish/v1/Chassis/{chassis}/NetworkAdapters/{function}
+  - Path: /redfish/v1/Chassis/{chassis}/NetworkAdapters/{function}/NetworkDeviceFunctions/{nic}
+  - Path: /redfish/v1/Chassis/{chassis}/NetworkAdapters/{function}/NetworkPorts/{port}
+
+  - Path: /redfish/v1/Chassis/{chassis}/PCIeSlots
+  - Path: /redfish/v1/Chassis/{chassis}/Power
+  - Path: /redfish/v1/Chassis/{chassis}/Sensors/{sensor}
+  - Path: /redfish/v1/Chassis/{chassis}/Thermal
+  # Added for v1.17
+  - Path: /redfish/v1/Chassis/{chassis}/PCIeDevices/{device}
+  - Path: /redfish/v1/Chassis/{chassis}/PCIeDevices/{device}/PCIeDevices/{function}
+  - Path: /redfish/v1/Chassis/{chassis}/NetworkAdapters/{nic}/Ports/{port}
```

Comment:
This new path is similar to `/redfish/v1/Chassis/{chassis}/NetworkAdapters/{nic}/NetworkPorts/{port}`.
If the path with `NetworkPorts` is not used in v1.17, please remove that rule.

It would be nice to place this line near other `NetworkAdapters` paths by sorting rules alphabetically.
We don't need to group lines added for v1.17.
We don't even need `Added for v1.17` comment line because we can see the difference with `diff` command.

Reply:
There are many NetworkPorts in collector generated JSON file. Then we can"t remove it.
I sorted alphabetically and remove comment "v1.17"

Reply:
Both `NetworkPorts` and `Ports` seem to be active.